app/services/claims/mentor/can_remove_from_claim.rb

```diff
@@ -0,0 +1,19 @@
+class Claims::Mentor::CanRemoveFromClaim
+  include ServicePattern
+
+  ACTIVE_CLAIM_STATUSES = %w[draft submitted].freeze
+
+  def initialize(mentor:)
+    @mentor = mentor
+  end
+
+  def call
+    mentor.mentor_trainings.each do |mentor_training|
+      claim_status = Claims::Claim.find(mentor_training.claim_id).status
+      return false if ACTIVE_CLAIM_STATUSES.include?(claim_status)
+    end
+    true
+  end
```

Comment:
This can potentially hit the database quite a few times. I'd advocate for minimal database queries as possible.

Maybe something like:

```ruby
Claims::Claim.active.joins(:mentor_trainings).where(mentor_trainings: { mentor_id: mentor }).exists?
```